### PR TITLE
Add attic date query to example queries

### LINKED ISSUE #259: Add attic date query to example queries
For inspecting vandalism or similar, attic date queries are helpful. For a quick access, it would be handy to have one in the example queries.

Recently, I used this one:

```
[date:"2016-01-01T00:00:00Z"]
[out:json][timeout:25];
(
  node({{bbox}});
  way({{bbox}});
  relation({{bbox}});
);
out body;
>;
out skel qt;
```

### PR DESCRIPTION
**Pull request by simon04**
Fixes #259.

### CODE REVIEW

**Comment by tyrasd** [comment]
thanks!

File: js/settings.js [hunk 1]
```diff
@@ -299,4 +299,23 @@ settings.define_upgrade_callback(32, function(s) {
...
-});
\ No newline at end of file
+});
+
+settings.define_upgrade_callback(33, function(s) {
+  s.saves["Attic date query"] = {
+    type: "example",
+    overpass: [
+      '/* This query loads all objects as of 2016-01-01 */',
+      '[date:"2016-01-01T00:00:00Z"]',
```
Comment (mmd-osm) [inline js/settings.js]: Should read: '[date:"2016-01-01T00:00:00Z"];',

Comment (tyrasd) [inline js/settings.js]: oops, totally overlooked that one. Will publish a fix in a minute.

Comment (tyrasd) [inline js/settings.js]: fixed in 1b2033bf9dac04b1ef8dea74302a67252fd5104e and published on overpass-turbo.eu

**Pull request merged**